### ISSUE TITLE
Site crashes when plugin installed

### ISSUE THREAD

**Issue by vbelolapotkov** [woocommerce/woocommerce-beta-tester#103]
### Error message
```
Parse error: syntax error, unexpected 'const' (T_CONST), expecting variable (T_VARIABLE) in /srv/users/usereb5bb9ff/apps/usereb5bb9ff/public/wp-content/plugins/woocommerce-beta-tester/includes/class-wc-beta-tester-import-export.php on line 17
There has been a critical error on this website. Please check your site admin email inbox for instructions.
```

![image](https://user-images.githubusercontent.com/3139099/134698508-b7c5f5b1-7a54-473d-908f-17495f3721b1.png)

### Environment details

* PHP 7.0
* WP 5.6.5
* WC 5.7.1

**Comment by rrennick** [comment]
@vbelolapotkov Which version of Woo beta tester?

**Comment by vbelolapotkov** [comment]
@rrennick , sorry, it was the latest from .org which is 2.0.3 IIRC.

**Comment by vbelolapotkov** [comment]
FYI, I experienced it in JN, so maybe it's JN specific, but I doubt. Similar issue we had with internal WC Pay dev tools with only difference it didn't crash the site completely. So my guess it's PHP version compatibility issue.

**Comment by rrennick** [comment]
> FYI, I experienced it in JN, so maybe it's JN specific

It is specific to JN. We're currently using the soon to be released 2.0.4 to test updates to Woorelease. Once 2.0.4 is released I'll do a PR to update JN. For now, you can build a copy from `trunk`. I'll leave this issue open until JN has been updated.

**Comment by rrennick** [comment]
PR to address this: https://github.com/Automattic/jurassic.ninja/pull/250
